nix/project.nix

```diff
@@ -0,0 +1,160 @@
+# editorconfig-checker-disable-file 
+
+{ repoRoot, inputs, lib, ... }:
+
+let
+  cabalProjectArgs = { config, pkgs, ... }: {
+
+    name = "plutus";
+
+    compiler-nix-name = lib.mkDefault "ghc92";
```

Comment:
If we're not merging the option sets, which I assume we aren't, then I guess this mkDefault does nothing?

Reply:
Or does it mean we can override it later in the variants? worth a comment

Reply:
haskell.nix complains if I don't do that. I'll add a comment